core/backend/src/IModelDb.ts

```diff
@@ -2747,6 +2748,8 @@ export class StandaloneDb extends BriefcaseDb {
     const nativeDb = this.openDgnDb(file, openMode, undefined, options);
 
     try {
+      if (openMode === OpenMode.ReadWrite)
```

Comment:
why is this necessary? I don't think we should do it. Perhaps you need to modify an existing test iModel?

Reply:
Our test copies a seed iModel and calls `StandaloneDb.openFile` to modify the copy.

Reply:
what seed iModel? It should have WAL mode enabled. If it's from git, I thought i updated it. Is there another one?

Reply:
The seed is delivered with the tests and doesn't have WAL mode enabled. 

I now see what you mean - if we always enable WAL when creating iModels, then we don't need to do that when opening them. But what about "legacy" ones, still using the DELETE mode - shouldn't we "upgrade" them to WAL too?

Reply:
that's why i added enabling WAL to the "SetToStandalone" tool. For tests, you can just run that once and check the file into git.

Reply:
Our team doesn't build/use the `display-test-app`, I wasn't aware the internal script was intended for general use rather than specific use of the test app. 

I converted the seed to WAL mode and am closing this PR.